doc/source/reference/c-api.array.rst

```diff
@@ -1671,10 +1671,11 @@ Conversion
 .. c:function:: PyObject* PyArray_GetField( \
         PyArrayObject* self, PyArray_Descr* dtype, int offset)
 
-    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>` (*self*, *dtype*, *offset*). Return
-    a new array of the given *dtype* using the data in the current
-    array at a specified *offset* in bytes. The *offset* plus the
-    itemsize of the new array type must be less than *self*
+    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>` (*self*, *dtype*, *offset*). This
+    function ``steals a reference <https://docs.python.org/3/c-api/intro.html?reference-count-details>``
+    to `PyArray_Descr` and returns a new array of the given `dtype` using
+    the data in the current array at a specified `offset` in bytes. The
+    `offset` plus the itemsize of the new array type must be less than `self`
     ->descr->elsize or an error is raised. The same shape and strides
```

Comment:
Needs reformatting to 80 columns.

I am not sure about the double backquotes around "steals a reference", that makes it format as code. I think you can just leave them out, since that phrase is known to developers familiar with the python C-API.

The last bit should be formatted as code ``self->decr->elsize``